README.md

```diff
@@ -263,6 +263,22 @@ We also have [a contributing guide](CONTRIBUTING.md).
 | **Supported Platforms (ARM)**     | macOS, Linux                                              |
 | **Configuration**                 | YAML                                                      |
 
+### Unstable functionality
+
+Some parts of the public API are marked as **unstable**.
+You can recognize this functionality from the warning in the API reference, or from the warning issued when the configuration option `warn_unstable` is active.
```

Comment:
I don't see the mentioned `warn_unstable` configuration option anywhere.

Reply:
I have removed this part sorry!